string objectName
string furnitureName
---
bool answer
